### PR TITLE
Visitor can navigate

### PR DESCRIPTION
**Pull request by julieahlquist**
````
As a visitor,
In order to login,
I would like to have a navbar with appropriate links.
````
Link to Pivotal Tracker: https://www.pivotaltracker.com/story/show/167833478

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,13 +6,15 @@
     "axios": "^0.19.0",
     "react": "^16.9.0",
     "react-dom": "^16.9.0",
-    "semantic-ui-css": "^2.4.1",
-    "semantic-ui-react": "^0.87.3",
+    "react-flash-message": "^1.0.2",
     "react-router-dom": "^5.0.1",
+    "react-timeout": "^1.2.0",
```
Comment (oliverochman) [inline package.json]: please remove this package

File: src/components/SignupForm.js [hunk 2]
```diff
@@ -4,19 +4,22 @@ import { Container, Form, Button } from "semantic-ui-react";
...
     name: "",
     email: "",
     password: "",
     password_confirmation: "",
-    userSaved: false,
-    errorMessage: ""
+    userSaved: false
   };
 
+  componentDidMount() {
```
Comment (oliverochman) [inline src/components/SignupForm.js]: please remove this

File: src/components/SignupForm.js [hunk 3]
```diff
@@ -172,7 +158,16 @@ const mapStateToProps = state => {
...
 
+const mapDispatchToProps = {
+  dispatchFlash: (error) => (
+    { type: 'SHOW_FLASH_MESSAGE', payload: { flashMessage: error.response.data.message, status: 'error' } }
+  ),
+  registerUser
+}
+
+
+
```
Comment (oliverochman) [inline src/components/SignupForm.js]: Remove empty lines

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (julieahlquist) [reply package.json]: fixed

File: src/components/SignupForm.js (hunk 2 above)
Comment (julieahlquist) [reply src/components/SignupForm.js]: fixed

File: src/components/SignupForm.js (hunk 3 above)
Comment (julieahlquist) [reply src/components/SignupForm.js]: fixed

**Review by oliverochman — APPROVED** [review]

**Pull request merged**